### DISCUSSION TITLE
No EDU email address

### DISCUSSION THREAD

**Discussion by startracex** [community/community#59833] [GitHub Education]
My school did not provide an EDU email address, I applied with my personal email address last year and could not continue to use it this year.

**Comment by CoolGenius-123** [comment]
Same thing happens to me as well not able to do the reverification of my student without college Email.

**Comment by D7x7z49** [comment]
I also have the same problem 😫

@startracex 👇
https://github.com/orgs/community/discussions/59360

**Comment by samus-aran** [comment]
Hi @startracex  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

Hope this helps! Let us know how you get along 👍
